ingest/pipeline/definition.json

```diff
@@ -77,5 +82,22 @@
       }
     ]
   }
+},
+{
+  "id": "apm_span",
+  "body": {
+    "description": "Removes span fields usually not needed to save storage",
```

Comment:
nit: I would go a bit more into detail which fields and why it is not needed to store them, e.g. something like `Removes detailed metadata information which is available on the span's transaction, to save storage`.